packages/scrubber-bar/src/scrubber-bar.ts

```diff
@@ -0,0 +1,207 @@
+import {
+  LitElement,
+  html,
+  css,
+  customElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+  CSSResult,
+} from 'lit-element';
+
+@customElement('scrubber-bar')
+export default class ScrubberBar extends LitElement {
+  @property({ type: Number }) value = 0;
+
+  @property({ type: Number }) min = 0;
+
+  @property({ type: Number }) max = 100;
+
+  @property({ type: Number }) step = 0.1;
+
+  get percentage(): number {
+    const delta: number = this.max - this.min;
+    const minOffset: number = this._value - this.min;
+    return (minOffset / delta) * 100;
+  }
+
+  private _userInteracting = false;
```

Comment:
It looks like there's a [long-standing conversation](https://github.com/microsoft/TypeScript/issues/9950) going on about that. The mindset at the moment seems to be since only Chrome and Opera support it right now, they weren't considering it its final form, even though they acknowledge, it'll likely be similar to that. 

So right now, no you have to use `private` syntax.

On a side note.. the hash symbol seems like such bad syntax to mark private. I'm curious how they landed on that. 😦